presto-base-arrow-flight/src/main/java/com/facebook/plugin/arrow/AbstractArrowSplitManager.java

```diff
@@ -46,7 +47,7 @@ public ConnectorSplitSource getSplits(ConnectorTransactionHandle transactionHand
     {
         ArrowTableLayoutHandle tableLayoutHandle = (ArrowTableLayoutHandle) layout;
         ArrowTableHandle tableHandle = tableLayoutHandle.getTableHandle();
-        FlightDescriptor flightDescriptor = getFlightDescriptor(clientHandler.getConfig(),
+        FlightDescriptor flightDescriptor = getFlightDescriptor(arrowFlightConfig,
```

Comment:
no need to have the `ArrowFlightConfig` parameter for `getFlightDescriptor`

Reply:
changed